doc/site/README.md

```diff
@@ -1,31 +1,56 @@
-# Recoil Site
+# Setting Up Local Site Development
```

Comment:
This title should be a subsection, not the main section (what is this? it's the site that powers recoil official page)